### PR TITLE
Point to Azure Spring Cloud documentation from Spring Monitor

### PR DESCRIPTION
**Pull request by jeanbisutti**

### CODE REVIEW

File: sdk/spring/spring-cloud-azure-starter-monitor/README.md [hunk 1]
```diff
@@ -28,43 +28,12 @@ For more information, please read [introduction to Application Insights][applica
 ```
 [//]: # ({x-version-update-end})
 
-#### Disable the JAR signature verification for the native image generation
+#### Azure Spring Cloud set-up
 
-You have to disable the JAR signature verification to be able to generate a native image ([see](https://github.com/Azure/azure-sdk-for-java/issues/30320
-)).
+You have to apply the Azure Spring Cloud set-up described [here](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/spring/README.md#spring-aot-and-spring-native-images) for Spring native images.
```
Comment (trask) [inline sdk/spring/spring-cloud-azure-starter-monitor/README.md:33]: I think calling it "Spring Cloud" is potentially confusing

File: sdk/spring/spring-cloud-azure-starter-monitor/README.md [hunk 2]
```diff
@@ -28,43 +28,12 @@ For more information, please read [introduction to Application Insights][applica
 ```
 [//]: # ({x-version-update-end})
 
-#### Disable the JAR signature verification for the native image generation
+#### Azure Spring Cloud set-up
```
Comment (heyams) [inline sdk/spring/spring-cloud-azure-starter-monitor/README.md:31]: how about "Spring Boot 3 Support"?

File: sdk/spring/spring-cloud-azure-starter-monitor/README.md (hunk 1 above)
Comment (heyams) [inline sdk/spring/spring-cloud-azure-starter-monitor/README.md:33]:
```suggestion
[Instruction](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/spring/README.md#spring-aot-and-spring-native-images) for Spring native images.
```

File: sdk/spring/spring-cloud-azure-starter-monitor/README.md [hunk 3]
```diff
@@ -28,43 +28,12 @@ For more information, please read [introduction to Application Insights][applica
...
 
-You have to disable the JAR signature verification to be able to generate a native image ([see](https://github.com/Azure/azure-sdk-for-java/issues/30320
-)).
+You have to apply the Azure Spring Cloud set-up described [here](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/spring/README.md#spring-aot-and-spring-native-images) for Spring native images.
 
-You can do this in the following way for GraalVM Native Build Tools:
+#### OpenTelemetry version set-up
```
Comment (heyams) [inline sdk/spring/spring-cloud-azure-starter-monitor/README.md:35]:
```suggestion
#### OpenTelemetry version adjustment
```

File: sdk/spring/spring-cloud-azure-starter-monitor/README.md (hunk 1 above)
Comment (jeanbisutti) [reply sdk/spring/spring-cloud-azure-starter-monitor/README.md:33]: "Spring Cloud Azure set-up"?

https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/spring/README.md#spring-aot-and-spring-native-images

image

**Review by ramthi — APPROVED** [review]

File: sdk/spring/spring-cloud-azure-starter-monitor/README.md (hunk 1 above)
Comment (jeanbisutti) [reply sdk/spring/spring-cloud-azure-starter-monitor/README.md]: I have changed for `Spring Cloud Azure set-up`

Comment (trask) [reply sdk/spring/spring-cloud-azure-starter-monitor/README.md]: my worry is that someone might think "I'm not using "Spring Cloud Azure" and skip over this section.

how about `Required native image build configuration` or something like that?

Comment (jeanbisutti) [reply sdk/spring/spring-cloud-azure-starter-monitor/README.md]: > Required native image build configurati

Done

**Review by heyams — APPROVED** [review]

**Pull request merged**